tool/generate_font.dart

```diff
@@ -120,24 +123,50 @@ void main(List<String> arguments) {
   output.writeAsStringSync(generatedOutput.join('\n'));
 }
 
-String generateIconDefinition(String iconName, String style, String unicode) {
+String generateIconDocumentation(
+    String iconName, String style, List searchTerms) {
+  searchTerms = searchTerms ?? [];
+  var searchTermsString = searchTerms.toString();
+  searchTermsString =
+      searchTermsString.substring(1, searchTermsString.length - 1);
+
+  iconName = iconName.replaceFirst("solid_", "");
```

Comment:
Would we need to take into account any other styles for the pro icon sets such as `light_` or `duotone_`?

Reply:
They deviated from their naming scheme for some solid icons only. We could add preemptive checks for other styles, but I don't see the need for that

Reply:
Sounds good!